HubSpot.NET/Core/Interfaces/IHubSpotApi.cs

```diff
@@ -8,6 +8,6 @@ public interface IHubSpotApi
         IHubSpotEngagementApi Engagement { get; }
         IHubSpotCosFileApi File { get; }
         IHubSpotOwnerApi Owner { get; }
-        IHubSpotCompanyPropertiesApi CompanyProperties { get; }
+        //IHubSpotCompanyPropertiesApi CompanyProperties { get; }
```

Comment:
Was there an issue with the Company Properties API? I only recently pushed that to Github so if you were missing files you might just need to sync up?

Reply:
Oops. I left that out by mistake